.circleci/config.yml

```diff
@@ -110,11 +110,15 @@ jobs:
       - run:
           name: Install CLI
           command: |
-            curl https://raw.githubusercontent.com/fossas/fossa-cli/$CIRCLE_SHA1/install.sh | bash
```

Comment:
`$CIRCLE_SHA1` accurately retrieves the current branch's `install.sh` script, however this does not make a difference because `install.sh` references latest tag. The focus here is to test the current binary which can be built using `make build`.